includes/widgets/testOne.php

```diff
@@ -0,0 +1,289 @@
+		
+		$this->start_controls_section(
+			'content_section',
+			[
+				'label' => esc_html__( 'table content', 'textdomain' ),
+				'tab' => \Elementor\Controls_Manager::TAB_CONTENT,
+			]
+		);
+
+		$this->add_control(
+			'show_title',
+			[
+				'label' => esc_html__( 'Show Columns Titles', 'textdomain' ),
+				'type' => \Elementor\Controls_Manager::SWITCHER,
+				'label_on' => esc_html__( 'Show', 'textdomain' ),
+				'label_off' => esc_html__( 'Hide', 'textdomain' ),
+				'return_value' => 'yes',
+				'default' => 'yes',
+			]
+		);
+
+		$this->add_control(
+			'columnOne',
+			[
+				'label' => esc_html__( 'Title column 1', 'elementor' ),
+				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
+				'placeholder' => esc_html__( 'Enter column title', 'elementor' ),
+				'default' => esc_html__( 'Column title', 'elementor' ),
+				'dynamic' => [
+					'active' => true,
+				],
+			]
+		);
+
+		$this->add_control(
+			'columnTwo',
+			[
+				'label' => esc_html__( 'Title column 2', 'elementor' ),
+				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
+				'placeholder' => esc_html__( 'Enter column title', 'elementor' ),
+				'default' => esc_html__( 'Column title', 'elementor' ),
+				'dynamic' => [
+					'active' => true,
+				],
+			]
+		);
+		$this->add_control(
+			'columnThree',
+			[
+				'label' => esc_html__( 'Title column 3', 'elementor' ),
+				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
+				'placeholder' => esc_html__( 'enter column title', 'elementor' ),
+				'default' => esc_html__( 'Column title', 'elementor' ),
+				'dynamic' => [
+					'active' => true,
+				],
+			]
+		);
+
+		$repeater = new repeater();
+
+		$repeater->add_control(
+			'text',
+			[
+				'label' => esc_html__( 'Text', 'elementor' ),
+				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
+				'placeholder' => esc_html__( 'List Item', 'elementor' ),
+				'default' => esc_html__( 'List Item', 'elementor' ),
+				'dynamic' => [
+					'active' => true,
+				],
+			]
+		);
+
+		$this->add_control(
+			'list',
+			[
+				'label' => esc_html__( 'Columns contents', 'textdomain' ),
+				'show_label' => esc_html__( true, 'textdomain' ),
+				'type' => \Elementor\Controls_Manager::REPEATER,
+				'fields' => $repeater->get_controls(),
+				'default' => [
+					[
+						'list_title' => esc_html__( 'Content column 1', 'textdomain' ),
+						'list_content' => esc_html__( 'Item content. Click the edit button to change this text.', 'textdomain' ),
+					],
+					[
+						'list_title' => esc_html__( 'Content column 2', 'textdomain' ),
+						'list_content' => esc_html__( 'Item content. Click the edit button to change this text.', 'textdomain' ),
+					],
+					[
+						'list_title' => esc_html__( 'Content column 3', 'textdomain' ),
+						'list_content' => esc_html__( 'Item content. Click the edit button to change this text.', 'textdomain' ),
+					],
+				],
+				'title_field' => '{{{ list_title }}}',
```

Comment:
Can you comment out this line? This line seems to be causing a problem. When I remove it, the repeater Add Item button works on my side.

@Mchris-hub

Reply:
i removed it